pyposeidon/schism.py

```diff
@@ -705,7 +714,12 @@ def save(self, **kwargs):
                 dic.update({"meteo": [x.attrs for x in meteo.Dataset]})
 
         coastlines = self.__dict__.get("coastlines", None)
-        dic.update({"coastlines": coastlines})
+        coastlines_database = os.path.join(path, "coastlines.json")
+        if coastlines is None:
+            dic.update({"coastlines": coastlines})
+        else:
+            coastlines.to_file(coastlines_database)
+            dic.update({"coastlines": coastlines_database})
```

Comment:
Maybe this is a bit more readable:
``` python
coastlines = self.__dict__.get("coastlines", None)
if coastlines:
    # Save the path to the serialized coastlines - #130
    coastlines_database = os.path.join(path, "coastlines.json")
    coastlines.to_file(coastlines_database)
    dic.update({"coastlines": coastlines_database}
else:
    dic.update({"coastlines": None})
```
Adding a comment with the ticket number is often useful for providing extra context.

Reply:
I need to use 
```python
if coastlines is not None:
```

because they can also be a dataframe and then I get 

```python
ValueError: The truth value of a GeoDataFrame is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all().
```